Specs/DateTime/Japanese/DatePeriodParser.json

```diff
@@ -4567,7 +4567,7 @@
         "Text": "08月から12月まで",
         "Type": "daterange",
         "Value": {
-          "Timex": "(XXXX-08-01,XXXX-12-01,P4M)",
+          "Timex": "(2018-08-01,2018-12-01,P4M)",
```

Comment:
The input doesn't specify the year, so ideally the timex should say XXXX. Is this an issue also in Chinese?
We should open a bug to fix it.

Reply:
yes that is what happens in general, but in the Chinese original parser code the year is assigned in certain cases (like if the reference date is within the range), so I thought it was intended.

Reply:
I will open the bug

Reply:
@tellarin, but this is also the behavior in English and the other languages. For an input like " from August to December", the timex will specify the year if the reference date is within the range and will use XXXX otherwise. For example in English:
"ReferenceDateTime": "2018-09-01T00:00:00" -----> "Timex": "(2018-08-01,2018-12-01,P4M)"
"ReferenceDateTime": "2018-07-01T00:00:00" -----> "Timex": "(XXXX-08-01,XXXX-12-01,P4M)"

Reply:
Yes, this is a bug that went overlooked. We should fix it.
The design is that a timex should be re-resolvable by passing a different reference datetime. In this case it would not work as the reference year is mistakenly hard-coded into the timex.

Reply:
I see, I will open a bug then.